src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -516,12 +516,12 @@ public object PrivateData
         public string[] ExternalModuleDependencies { get; set; }
 
         /// <summary>
-        /// Specify the HelpInfo URI.
+        /// Gets or sets the HelpInfo URI.
         /// </summary>
         [Parameter]
         [AllowNull]
         [SuppressMessage("Microsoft.Design", "CA1056:UriPropertiesShouldNotBeStrings")]
-        public string HelpInfoUri
+        public string HelpInfoURI
```

Comment:
Please revert - it is public API (breaking change).

Reply:
done